python/composio/client/http.py

```diff
@@ -18,57 +18,6 @@
 DEFAULT_REQUEST_TIMEOUT = 60.0
 
 
```

Comment:
Great job on removing the unused `AsyncHttpClient` class. This cleanup will help keep the codebase more maintainable and reduce potential confusion for other developers.